packages/core/src/components/search/Filter/FilterSlider.tsx

```diff
@@ -140,9 +220,19 @@ function FilterSlider({
         onClick: () => {
           resetInfiniteScroll(0)
 
+          const noPickupInPointShippingFacet = selected.find(
```

Comment:
I think I didn't understand this part 😕

Reply:
I think it'll be clear change to `!hasPickupInPointShippingFacet`, it's basically that 😬